### PR TITLE
Remove Developers section

### PR DESCRIPTION
**Pull request by Mariatta**
Developers section has been moved to https://api.coala.io
remove developers section from table of contents
see https://github.com/coala/coala/pull/3134

Closes https://github.com/coala/documentation/issues/336

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -146,3 +146,7 @@ rely on the upcoming release being rock stable.
 If you want to see how the development progresses, check out
 `coala <https://github.com/coala/coala>`__ and
 `coala-bears <https://github.com/coala/coala-bears>`__.
+
+
+.. _tutorial for newcomers: https://api.coala.io/en/latest/Developers/Newcomers_Guide.html
```
Comment (sils) [inline index.rst]: use coala.io/newcomers , I already updated all shortlinks

File: index.rst [hunk 2]
```diff
@@ -146,3 +146,7 @@ rely on the upcoming release being rock stable.
...
+.. _Writing Bears: https://api.coala.io/en/latest/Developers/Writing_Bears.html
```
Comment (sils) [inline index.rst]: coala.io/writingbears

**Comment by sils** [comment]
otherwise good :) happy to have this separated :)

**Comment by Mariatta** [comment]
Thanks 😃 I updated the links to coala.io/newcomers and coala.io/writingbears

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/340/commits/7940f15aba70161531c0e440b60af782cb03886b

**Comment by sils** [comment]
@rultor merge

**Pull request merged**